src/templates/ratings/add.html

```diff
@@ -20,7 +26,7 @@
         </select>
       </p>
       <textarea id="review-body" rows="2" cols="40" name="body"
-                placeholder="{{ _('Tell us what you love about this app!') }}"
+                placeholder="{{ _('Tell us what you like about this app. Write like you’re telling a friend about your experience with the app. Give specific details, for example, What features did you like or dislike?') }}"
```

Comment:
@gurumukhi can we remove this change? I know the bug says it, but I think it's a little bit rambling